src/doc/rustdoc/src/command-line-arguments.md

```diff
@@ -100,7 +100,8 @@ mod private { // this item is private and will not be documented
 }
 ```
 
-`--document-private-items` documents all items, even if they're not public.
+`--document-private-items` includes all non-public in the generated documentation except for `#[doc(hidden)]` items.
```

Comment:
Please mention that there will be a visual marker to indicate that these items are private.